### DISCUSSION TITLE
how to use clear selected event

### DISCUSSION THREAD

**Discussion by MohsenAfshari** [sagalbot/vue-select#1627] [Help Wanted]
i want to use when click clear selected fire event

**Comment by mininvn** [comment]
In case you want to do something when the value is set to `undefined` by the `clear selected` button, I think you can use `@update:modelValue` event. I'm using version 4.0.0-beta.5 and got a problem is that `@option:deselected` didn't fire correctly and `@option:selected` won't fire when I clicked the `clear selected` button. `@update:modelValue` will fire whenever the value changes so it fits my case. https://github.com/sagalbot/vue-select/issues/1728
